src/warpychain/eth_client.go

```diff
@@ -0,0 +1,31 @@
+package warpychain
+
+import (
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/sirupsen/logrus"
+)
+
+type EthUrl int64
+
+const (
+	Avax EthUrl = iota
+)
+
+func (ethUrl EthUrl) String() string {
+	switch ethUrl {
+	case Avax:
+		return "https://api.avax.network/ext/bc/C/rpc"
+	}
+
+	return "unknown"
```

Comment:
hmm, what is the point of returning 'unknown' here? shouldn't an explicit error be returned here?

Reply:
btw. @janekolszak is there a way to 'natively' implement exhaustive switch in Go? I've only found this: https://pkg.go.dev/github.com/nishanths/exhaustive